packages/experiments/src/components/Card/examples/Card.Basic.Example.tsx

```diff
@@ -0,0 +1,57 @@
+// @codepen
+import * as React from 'react';
+import { Card } from '../Card';
+import { Persona } from '../../../Persona';
+import { PersonaTestImages } from '@uifabric/experiments/lib/common/TestImages';
+import { Stack } from '../../../Stack';
+import { Text } from '../../../Text';
+import { Icon, Image } from 'office-ui-fabric-react';
+import { mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
+
+export class CardBasicExample extends React.Component<{}, {}> {
+  public render(): JSX.Element {
+    const styles = mergeStyleSets({
+      textLarge: {
+        color: 'teal',
+        fontWeight: 'bolder'
+      },
+      textBold: {
+        fontWeight: 'bold'
+      }
+    });
+
+    return (
+      <Stack gap={5}>
+        <Card>
+          <Persona text="Kevin Jameson" secondaryText="Feb 2, 2019" coin={{ imageUrl: PersonaTestImages.personMale }} />
+          <Card.Item preventPadding>
+            <Image
+              src="https://placehold.it/250x120"
+              alt="Example implementation of the property image fit using the center value on an image larger than the frame."
+              width="100%"
+              height="120px"
+            />
+          </Card.Item>
+          <Text variant="large" className={styles.textLarge}>
+            Contoso
+          </Text>
+          <Text wrap variant="large">
+            Contoso Denver expansion design marketing hero guidelines
+          </Text>
+          <Text className={styles.textBold}>Is this recommendation helpful?</Text>
+          <Card.Item preventPadding>
```

Comment:
I'm curious if `preventPadding` is used in 2 out of 3 cases that it should instead be inverted and applied by default? `enablePadding`?